parsec/backend/postgresql/vlob_queries/write.py

```diff
@@ -0,0 +1,162 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pendulum
+from triopg import UniqueViolationError
+from uuid import UUID
+
+from parsec.api.protocol import DeviceID, OrganizationID
+from parsec.backend.postgresql.utils import (
+    Q,
+    query,
+    q_organization_internal_id,
+    q_device_internal_id,
+    q_realm_internal_id,
+    q_vlob_encryption_revision_internal_id,
+)
+from parsec.backend.vlob import VlobVersionError, VlobTimestampError, VlobNotFoundError
+from parsec.backend.postgresql.handler import send_signal
+from parsec.backend.postgresql.vlob_queries.utils import query_get_realm_id_from_vlob_id
+from parsec.backend.backend_events import BackendEvent
+
+
+q_vlob_updated = Q(
+    f"""
+INSERT INTO realm_vlob_update (
+realm, index, vlob_atom
+)
+SELECT
+{ q_realm_internal_id(organization_id="$organization_id", realm_id="$realm_id") },
+(
+    SELECT COALESCE(MAX(index) + 1, 1)
+    FROM realm_vlob_update
+    WHERE realm = { q_realm_internal_id(organization_id="$organization_id", realm_id="$realm_id") }
+),
+$vlob_atom_internal_id
+RETURNING index
+"""
+)
+
+
+@query()
+async def query_vlob_updated(
+    conn, vlob_atom_internal_id, organization_id, author, realm_id, src_id, src_version=1
+):
+
+    index = await conn.fetchval(
```

Comment:
a transaction is missing here (given were are doing a `fetchval`, then a `send_signal`)